.vscode/settings.json

```diff
@@ -1,11 +1,10 @@
 {
-    "editor.formatOnPaste": true,
-    "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode",
-    "editor.codeActionsOnSave": {
-      "source.fixAll.eslint": true,
-      "source.fixAll.format": true
-    },
-    "svg.preview.background": "editor"
-  }
-  
\ No newline at end of file
+  "editor.formatOnPaste": true,
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": "explicit",
+    "source.fixAll.format": "explicit"
+  },
+  "svg.preview.background": "editor"
+}
```

Comment:
This has to be added to the gitignore, as it is part of the vs code configuration.

Reply:
Done